### DISCUSSION TITLE
Redis只有密码认证的情况下，无法绑定，希望能支持一下

### DISCUSSION THREAD

**Discussion by JianhaoLu** [illacloud/illa-builder#631] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/25343841/205868134-c29e2a75-da02-4dbb-96b7-8e179673f2c9.png)

Another Redis Desktop Manager上可以只填写密码进行登录的（如下图）
![image](https://user-images.githubusercontent.com/25343841/205868200-98a9fb9b-66a7-4bc8-894f-43b71c9f420b.png)

**Comment by AruSeito** [comment, marked as answer]
您好，感谢您的反馈。预计本周四发版更新。
